### PR TITLE
fix:change edge_site to edgesite in edgesite small build

### PR DESCRIPTION
**Pull request by zhanglianx111**

### CODE REVIEW

File: edgesite/Makefile [hunk 1]
```diff
@@ -38,4 +38,4 @@ armv8: verify
 .PHONY: small_build
 small_build: verify
 	go build -ldflags="-w -s -extldflags -static" cmd/edgesite.go
-	upx-ucl --best edge_site
+	upx-ucl --best edgesite
```
Comment (fisherxu) [inline edgesite/Makefile]: Can we add the edgesite `small build` to travis CI？ @zhanglianx111

Comment (zhanglianx111) [reply edgesite/Makefile]: ok

File: .travis.yml [hunk 2]
```diff
@@ -49,6 +49,7 @@ jobs:
         - make edge_cross_build_v7
         - make edgesite_cross_build
         - make edgesite_cross_build_v7
+        - make edgestite_small_build
```
Comment (fisherxu) [inline .travis.yml]: typo here `edgestite_small_build`

**Pull request merged**